tools/signaturetest/src/main/java/com/sun/ts/tests/signaturetest/signature-repository/jakarta.annotation.sig

```diff
@@ -1,5 +1,5 @@
 #Signature file v4.1
-#Version 2.1
+#Version 1.0.9
```

Comment:
Might not be critical, but is this right? Seems strange to me :)

Reply:
fixed!